src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -49,7 +49,7 @@
     height: 100%;
     justify-content: center;
     left: 0;
-    opacity: 1;
+    opacity: 0;
```

Comment:
Had to change this otherwise it looks like that it hangs when closing because of `transition: all 0.3s`